### PR TITLE
Muzima 398

### PR DESCRIPTION
**Pull request by mssavai**

### CODE REVIEW

File: src/main/java/com/muzima/api/service/impl/ObservationServiceImpl.java [hunk 1]
```diff
@@ -328,6 +338,16 @@ public Observation getObservationByUuid(final String uuid) throws IOException {
...
     }
+    /**
+     * {@inheritDoc}
+     *
+     * @see ObservationService#getObservationsByEncounter(String)
+     */
+    @Override
+    public List<Observation> getObservationsByEncounter(final String encounterUuid) throws IOException {
+        List<Observation> obs = observationDao.get(null, null ,encounterUuid);
+        return obs;
```
Comment (sthaiya) [inline src/main/java/com/muzima/api/service/impl/ObservationServiceImpl.java]: Can we change to `return observationDao.get(null, null ,encounterUuid);` remove the redundant variable?

File: src/main/java/com/muzima/api/dao/impl/ObservationDaoImpl.java [hunk 2]
```diff
@@ -46,7 +46,42 @@ protected ObservationDaoImpl() {
...
+        if (!StringUtil.isEmpty(conceptUuid)) {
+            Filter conceptFilter = FilterFactory.createFilter("conceptUuid", conceptUuid);
+            filters.add(conceptFilter);
+        }
+        if (!StringUtil.isEmpty(encounterUuid)) {
+            Filter conceptFilter = FilterFactory.createFilter("encounterUuid", encounterUuid);
+            filters.add(conceptFilter);
+        }
+        List<Observation> obs = service.getObjects(filters, daoClass);
+        return obs;
```
Comment (sthaiya) [inline src/main/java/com/muzima/api/dao/impl/ObservationDaoImpl.java]: remove the redundant return variable obs

File: src/main/java/com/muzima/api/dao/impl/ObservationDaoImpl.java [hunk 3]
```diff
@@ -46,7 +46,42 @@ protected ObservationDaoImpl() {
             Filter conceptFilter = FilterFactory.createFilter("conceptUuid", conceptUuid);
             filters.add(conceptFilter);
         }
-        return service.getObjects(filters, daoClass);
+        List<Observation> obs = service.getObjects(filters, daoClass);
```
Comment (sthaiya) [inline src/main/java/com/muzima/api/dao/impl/ObservationDaoImpl.java]: remove the redundant return variable obs

File: src/main/java/com/muzima/api/service/impl/ObservationServiceImpl.java (hunk 1 above)
Comment (mssavai) [inline src/main/java/com/muzima/api/service/impl/ObservationServiceImpl.java]: Updated :)

**Pull request merged**